admin/config.yml

```
@@ -0,0 +1,151 @@
+  # Gallery Collection
+  - name: "photo_gallery"
+    label: "Photo Gallery"
+    folder: "content/gallery"
+    create: true
+    slug: "{{year}}-{{month}}-{{slug}}"
+    fields:
+      - {label: "Album Title", name: "title", widget: "string"}
+      - {label: "Date", name: "date", widget: "date"}
+      - {label: "Description", name: "description", widget: "text", required: false}
+      - {label: "Photos", name: "photos", widget: "list", fields: [
+          {label: "Image", name: "image", widget: "image"},
```

YAMLlint (1.37.1)

[warning] wrong indentation: expected 8 but found 10

(indentation)